### DISCUSSION TITLE
How can I debug isAddtionalStep?

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [github/codeql#8464] [General]
I have a subclass of TaintTracking::Configuration. Both isSource and isSink works fine in quick evaluation. But the flow is somehow broken between them so I need to use isAddtionalStep to link the nodes.

How can I figure out which part needs to be glued together?

**Comment by smowton** [comment]
How about trying https://codeql.github.com/docs/writing-codeql-queries/debugging-data-flow-queries-using-partial-flow/
